codemeta.json

```diff
@@ -0,0 +1,43 @@
+{
+  "@context": "https://doi.org/10.5063/schema/codemeta-2.0",
+  "@type": "SoftwareSourceCode",
+  "license": "https://spdx.org/licenses/MIT",
+  "codeRepository": "https://github.com/openMetadataInitiative/openMINDS_MATLAB",
+  "contIntegration": "https://github.com/openMetadataInitiative/openMINDS_MATLAB/actions",
+  "issueTracker": "https://github.com/openMetadataInitiative/openMINDS_MATLAB/issues",
+  "name": "openMINDS MATLAB",
+  "description": "openMINDS MATLAB is a toolbox to support the creation and use of openMINDS metadata models and schemas in MATLAB, with import and export in JSON-LD format. The package contains all openMINDS schemas as MATLAB classes in addition to schema base classes and utility methods.",
+  "applicationCategory": "neuroscience",
+  "releaseNotes": "https://github.com/openMetadataInitiative/openMINDS_MATLAB/releases",
+  "funding": "https://cordis.europa.eu/project/id/945539",
+  "developmentStatus": "active",
+  "funder": {
+    "@type": "Organization",
+    "name": "European Commission"
+  },
+  "programmingLanguage": [
+    "MATLAB"
+  ],
+  "operatingSystem": [
+    "Linux",
+    "Windows",
+    "macOS"
+  ],
+  "softwareRequirements": [
+    "MATLAB R2022b or later"
+  ],
+  "relatedLink": [
```

Comment:
instead or in addition you could define isPartOf since this repo is part of the open metadata initiative organisation on github